client/src/components/ui/score-card.tsx

```diff
@@ -0,0 +1,27 @@
+export type Score = "high" | "medium" | "low";
+
+interface ScoreIndicatorProps {
+  score: Score;
+  className?: string;
+  children?: React.ReactNode;
+}
+
+export const ScoreIndicator = ({
+  score,
+  children,
+  className = "",
+}: ScoreIndicatorProps) => {
+  const bgColorClass = {
+    high: "bg-high",
+    medium: "bg-medium",
+    low: "bg-low",
+  }[score];
+
+  return (
+    <div
+      className={`flex h-10 items-center justify-center font-medium capitalize text-deep-ocean ${bgColorClass} ${className}`}
+    >
+      {children ? children : score}
```

Comment:
What's the difference between `children` and `score`? I think we can just use `children` here.

Reply:
Based on the figma, I believe we can either have the Score Union (low, medium and high) OR an integer currency value like ($123).
In case its an integer we must pass the value to the Scoreindicator as child which should render inside of the colored box and if its just an Union then we can render the value directly from the `score` props, So children is optional, What do you think ?

Reply:
I still see both things a `children`. The good thing of using `children` is that you can pass anything to the component and it will render. Any custom logic (like parsing the number) should be done outside and just pass the final outcome.

```
<ScoreCard>
  Low
</ScoreCard>

<ScoreCard>
  {parseNumber(value)}
</ScoreCard>
```

Reply:
What if the user passes some invalid value as children, once we remove the score prop ?
For example - 
```
<ScoreCard>
  Invalid string
</ScoreCard>
```

Even for below example the component doesnt know which colored box to render -
```
<ScoreCard>
  {parseNumber(value)}
</ScoreCard>
```

We need an additional prop like score or similar to know which box to render, right ?